src/Relude/Bool/Guard.hs

```diff
@@ -86,6 +88,7 @@ guardM f = f >>= guard
 
 -- | Monadic version of 'Data.Bool.(||)' operator.
 --
+-- >>> :set -Wno-deprecations
 -- >>> Just True ||^ undefined
```

Comment:
Thanks for the fix! Minor comment: I think it's either better to move `:set -Wno-deprecations` to the `$setup` part of the module or replce `undefined` with `error "Shouldn't be evaluated` or something like that. Having `:set -Wno-deprecations` in visible documentation is a little bit ugly and might make readers afraid of using `relude`...